macOS build fails with error cannot find GeneratedPluginRegistrant.swift:12:3

```
Building macOS application...
/Users/xxx/Sites/insta_sensors/macos/Flutter/GeneratedPluginRegistrant.swift:12:3: error: cannot find 'FlutterSecureStorageMacosPlugin' in scope
  FlutterSecureStorageMacosPlugin.register(with: registry.registrar(forPlugin: "FlutterSecureStorageMacosPlugin"))
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Am I doing something wrong or is it an issue with the latest XCode?

> I know it's not correct, but just to help temporarily, but just change from FlutterSecureStorageMacosPlugin to SwiftFlutterSecureStoragePlugin in the generated file in Flutter/GeneratedPluginRegistrant

It doesn't work for me. GeneratedPluginRegistrant.swift regenerate itself on build action and build fails.  :(

Launching lib/main.dart on macOS in debug mode...
lib/main.dart:1
/Users/gaston/Desktop/Proyecto_llaves/desktop/llavero/macos/Flutter/GeneratedPluginRegistrant.swift:11:3: error: cannot find 'FlutterSecureStorageMacosPlugin' in scope
  FlutterSecureStorageMacosPlugin.register(with: registry.registrar(forPlugin: "FlutterSecureStorageMacosPlugin"))
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
** BUILD FAILED **

Exception: Build process failed
Exited

Building macOS application...
[project]/macos/Flutter/GeneratedPluginRegistrant.swift:14:3: error: cannot find 'FlutterSecureStorageMacosPlugin' in scope
  FlutterSecureStorageMacosPlugin.register(with: registry.registrar(forPlugin: "FlutterSecureStorageMacosPlugin"))

--- a/flutter_secure_storage_macos/macos/Classes/FlutterSecureStoragePlugin.swift
+++ b/flutter_secure_storage_macos/macos/Classes/FlutterSecureStoragePlugin.swift
@@ -1,5 +1,5 @@
 //
-//  SwiftFlutterSecureStoragePlugin.swift
+//  FlutterSecureStoragePlugin.swift
 //  flutter_secure_storage_macos
 //
 //  Created by Julian Steenbakker on 09/12/2022.
@@ -7,13 +7,13 @@
 
 import FlutterMacOS
 
-public class SwiftFlutterSecureStoragePlugin: NSObject, FlutterPlugin {
+public class FlutterSecureStoragePlugin: NSObject, FlutterPlugin {
     
     private let flutterSecureStorageManager: FlutterSecureStorage = FlutterSecureStorage()
     
     public static func register(with registrar: FlutterPluginRegistrar) {
         let channel = FlutterMethodChannel(name: "plugins.it_nomads.com/flutter_secure_storage", binaryMessenger: registrar.messenger)
-        let instance = SwiftFlutterSecureStoragePlugin()
+        let instance = FlutterSecureStoragePlugin()
         registrar.addMethodCallDelegate(instance, channel: channel)
     }
     

--- a/flutter_secure_storage_macos/pubspec.yaml
+++ b/flutter_secure_storage_macos/pubspec.yaml
@@ -17,4 +17,4 @@ flutter:
     implements: flutter_secure_storage
     platforms:
       macos:
-        pluginClass: FlutterSecureStorageMacosPlugin
+        pluginClass: FlutterSecureStoragePlugin

